scripts/setup.sh

```diff
@@ -4,14 +4,12 @@ WHERE=$(cd `dirname ${0}` && pwd)
 cd ${WHERE}
 
 # Default value for VENV_PATH is bootstrap_venv
-VENV_PATH=${1:-bootstrap_venv}
+VENV_PATH="bootstrap_venv"
```

Comment:
Why is this a better? The original sets the path to the default unless the first argument overrides it.